Thu, 16 Jan 2020 12:12:15 - INFO - Connect to DB
Thu, 16 Jan 2020 12:12:15 - INFO - TRIZMA DB import
Thu, 16 Jan 2020 12:12:15 - INFO - 123TV DB import
Thu, 16 Jan 2020 12:14:07 - INFO - Connect to DB
Thu, 16 Jan 2020 12:14:08 - INFO - TRIZMA DB import
Thu, 16 Jan 2020 12:14:08 - INFO - 123TV DB import
Thu, 16 Jan 2020 12:16:30 - INFO - Connect to DB
Thu, 16 Jan 2020 12:16:30 - INFO - TRIZMA DB import
Thu, 16 Jan 2020 12:16:30 - INFO - 123TV DB import
Thu, 16 Jan 2020 12:17:12 - INFO - Connect to DB
Thu, 16 Jan 2020 12:17:12 - INFO - TRIZMA DB import
Thu, 16 Jan 2020 12:17:12 - INFO - 123TV DB import
Thu, 16 Jan 2020 12:25:19 - INFO - Connect to DB
Thu, 16 Jan 2020 12:25:20 - INFO - TRIZMA DB import
Thu, 16 Jan 2020 12:25:20 - INFO - Failed on TRIZMA DB import at: 2020-01-16_12-06-59, for following calling party: 4927642619321, for agent id: trz_bmaksimovic, with error:(pyodbc.DataError) ('22001', '[22001] [Microsoft][ODBC Driver 17 for SQL Server][SQL Server]String or binary data would be truncated. (8152) (SQLExecDirectW)')
[SQL: insert into records values('CC2Servicenummer','Belgrad','Nicht übermittelt','trz_bmaksimovic',                                 '4927642619321','','Nicht übermittelt','4927642619321_Thursday, January 16, 2020_1:25:16 PM','BOR_MAKSIM')]
(Background on this error at: http://sqlalche.me/e/9h9h)
Thu, 16 Jan 2020 12:25:20 - INFO - 123TV DB import
